src/Event/EventManager.php

```diff
@@ -442,6 +442,24 @@ public function prioritisedListeners($eventKey)
     }
 
     /**
+     * Returns the listeners matching a specified pattern
+     *
+     * @param string $eventKeyPattern Pattern to match.
+     * @return array
+     */
+    public function matchingListeners($eventKeyPattern)
+    {
+        $matchPattern = '/' . preg_quote($eventKeyPattern, "/") . '/';
```

Comment:
Indeed. So would you rather I just anchor the ``$eventKeyPattern`` and correct the unit-tests?